rpm/sample-app-c.spec

```diff
@@ -14,13 +14,11 @@ Simple app written in C
 %autosetup -p1 -n %{name}-%{version}
 
 %build
-cd src
-gcc -o sample-app-c sample-app-c.c -lssusysinfo
+make -C src
```

Comment:
Should be `%make_build`, but this is affected by jb#58377. So I would suggest to either 1) file a separate task for replacing make here and do it after next SDK release or 2) replace it now and take it as an opportunity to demonstrate the bug and how to deal with it during training.

Reply:
I think I'll take option 3) file a separate task and add a comment here now